pkg/crypto/tinkcrypto/primitive/secp256k1/secp256k1_signer_key_manager_test.go

```diff
@@ -278,14 +278,6 @@ func TestPublicKeyDataWithInvalidInput(t *testing.T) {
 
 	_, err = pkm.PublicKeyData(serializedKey)
 	require.Error(t, err, "expect an error when input is a modified serialized key")
-
-	// nil
-	_, err = pkm.PublicKeyData(nil)
-	require.Errorf(t, err, "expect an error when input is nil")
-
-	// empty slice
-	_, err = pkm.PublicKeyData([]byte{})
```

Comment:
No longer returns an error with the updated Tink version (pub key bytes in returned struct are just empty now)